ddtrace/opentracer/span_context.py

```diff
@@ -1,15 +1,37 @@
 from opentracing import SpanContext as OpenTracingSpanContext
 
 
+from ddtrace.context import Context
+
+
 class SpanContext(OpenTracingSpanContext):
-    """"""
+    """Implementation of the OpenTracing span context."""
 
-    __slots__ = []
+    def __init__(self, trace_id=None, span_id=None, sampled=True,
+                 sampling_priority=None,
+                 baggage=OpenTracingSpanContext.EMPTY_BAGGAGE, context=None):
```

Comment:
`baggage` is a mutable object. We should set the default in the function body otherwise we mutate a global instance. See the comment in #470 for more details.